SW_Output.c

```diff
@@ -1653,7 +1668,20 @@ void SW_OUT_set_colnames(void) {
 	if (debug) swprintf("SW_OUT_set_colnames: set columns for 'eSW_Temp' ...");
 	#endif
 	for (i = 0; i < ncol_OUT[eSW_Temp]; i++) {
-		colnames_OUT[eSW_Temp][i] = Str_Dup(cnames_eSW_Temp[i]);
+        if(i < 3) {
+            strcpy(ctemp, cnames_eSW_Temp[i]);
+        } else {
+            strcpy(ctemp, cnames_eSW_Temp[3]);
+            strcat(ctemp, "_");
+            if(i == 3) {

```

Comment:
Why are columns of surface temperature organized as `| avg | max | min |` (requiring this `if-else`) instead of the `| max | min | avg |` (as dictated by `*cnames_eSW_Temp[]`) that is used elsewhere, i.e., air temperature and for soil temperature? I suggest using the same order and then the `if-else` is no longer needed. This harmonization will also need to be reflected in the `get` output functions.